### DISCUSSION TITLE
Comments cannot be set with 'Reply' and 'Resolve'

### DISCUSSION THREAD

**Discussion by Aila0014** [dolanmiu/docx#1868] [Q&A] [answered]
I am working on a function of generating word from html. Similar to exporting rich text editor to word, my editor has the same comment function as word. You can 'reply' or check 'solved' below the comment, but the comment here does not have this detailed configuration. Do you have any good methods?

**Comment by dolanmiu** [comment, marked as answer]
I don't really understand the question

Sounds off topic

**Reply by Aila0014** [reply]
Another explanation is that word can reply to comments. I hope to add these parameters when generating documents

**Reply by dolanmiu** [reply]
Is this possible with Microsoft Word? Can you show me an example in Word?

**Reply by dolanmiu** [reply]
Replying is not possible.

I did some research, and it is a Microsoft Word only feature. This will not work in other Word Processors
